pmm/pmm2-server-release.groovy

```diff
@@ -107,77 +107,44 @@ pipeline {
-                     unstash 'copy'
-                     sh """
-                         echo ${VERSION} > VERSION
-                     """"
-                     sh """
-                         echo "/usr/bin/ssh -i "${SSHKEY}" -o StrictHostKeyChecking=no \\\"\\\$@\\\"" > github-ssh.sh
-                         chmod 755 github-ssh.sh
-                         git config --global user.email "dev-services@percona.com"
-                         git config --global user.name "PMM Jenkins"
-
-                     """
-                     sh '''
-                         export GIT_SSH=$(pwd -P)/github-ssh.sh
-                         export GPG_AGENT_INFO=${HOME}/.gnupg/S.gpg-agent
-                         export VERSION=$(cat VERSION)
-                         export TOP_VER=$(cat VERSION | cut -d. -f1)
-                         export MID_VER=$(cat VERSION | cut -d. -f2)
-                         export DOCKER_MID="$TOP_VER.$MID_VER"
-                         declare -A repo=(
-                             ["percona-dashboards"]="percona/grafana-dashboards"
-                             ["pmm"]="percona/pmm"
-                             ["pmm-admin"]="percona/pmm-admin"
-                             ["pmm-agent"]="percona/pmm-agent"
-                             ["pmm-server"]="percona/pmm-server"
-                             ["pmm-server-packaging"]="percona/pmm-server-packaging"
-                             ["percona-qan-app"]="percona/qan-app"
-                             ["percona-qan-api2"]="percona/qan-api2"
-                             ["pmm-update"]="percona/pmm-update"
-                             ["pmm-managed"]="percona/pmm-managed"
-                             ["pmm-qa"]="percona/pmm-qa"
-                         )
-
-                         for package in "${!repo[@]}"; do
-                             SHA=$(
-                                 grep "^$package-$VERSION-" copy.list \
-                                     | perl -p -e 's/.*[.]\\d{10}[.]([0-9a-f]{7})[.]el7.*/$1/'
-                             )
-                             if [[ -n "$package" ]] && [[ -n "$SHA" ]]; then
-                                 rm -fr $package
-                                 mkdir $package
-                                 pushd $package >/dev/null
-                                     git clone --recursive git@github.com:${repo["$package"]} ./
-                                     git checkout $SHA
-                                     git update submodules
-                                     git tag -s v$VERSION \
-                                             -m "Version $VERSION"
-                                     git push --tags
+                withCredentials([string(credentialsId: 'GITHUB_API_TOKEN', variable: 'GITHUB_API_TOKEN')]) {
+                    unstash 'copy'
+                    sh """
+                        echo ${GITHUB_API_TOKEN} > GITHUB_API_TOKEN
```

Comment:
it this case you can add secret file as a credentials in jenkins and use it as a file

Reply:
It's a revert PR for WIP branch. :)
Thanks for the hint, I'll fix this as soon as I'll proceed working on this ticket